mappings/net/minecraft/block/entity/LootableContainerBlockEntity.mapping

```diff
@@ -3,9 +3,9 @@ CLASS net/minecraft/class_2621 net/minecraft/block/entity/LootableContainerBlock
 	FIELD field_12037 lootTableId Lnet/minecraft/class_2960;
 	FIELD field_31352 LOOT_TABLE_SEED_KEY Ljava/lang/String;
 	FIELD field_31353 LOOT_TABLE_KEY Ljava/lang/String;
-	METHOD method_11281 setInvStackList (Lnet/minecraft/class_2371;)V
-		ARG 1 list
-	METHOD method_11282 getInvStackList ()Lnet/minecraft/class_2371;
+	METHOD method_11281 setInventory (Lnet/minecraft/class_2371;)V
+		ARG 1 inventory
+	METHOD method_11282 getInventory ()Lnet/minecraft/class_2371;
```

Comment:
Not sure if I'd call this "inventory" since it's just a `DefaultedList<ItemStack>` used to implement the `Inventory` (which is the `LootableContainerBlockEntity` itself)

Reply:
I reverted it for now. Maybe I'll look into it someday.